Sources/Onboarding/ViewModels/UnlockScreenViewModel.swift

```diff
@@ -0,0 +1,29 @@
+import GDSAnalytics
+import GDSCommon
+import Logging
+import UIKit
+
+struct UnlockScreenViewModel: BaseViewModel {
+    let rightBarButtonTitle: GDSLocalisedString? = nil
+    let backButtonIsHidden: Bool = true
+    let primaryButtonViewModel: ButtonViewModel
+    let analyticsService: AnalyticsService
+
+    init(analyticsService: AnalyticsService,
+         primaryButtonAction: @escaping () -> Void ) {
+        self.analyticsService = analyticsService
+        self.primaryButtonViewModel = AnalyticsButtonViewModel(titleKey: "app_unlockButton",
+                                                               analyticsService: analyticsService) {
+            primaryButtonAction()
+        }
+    }
+
+    func didAppear() {
+        let screen = ScreenView(screen: IntroAnalyticsScreen.returningBiometricUser, titleKey: primaryButtonViewModel.title.stringKey)
```

Comment:
I'm not sure we want to be populating this `ScreenView`'s title with the button title, understand there isn't a title on this screen but we should probably check with analytics what they would want here in the long run. Potentially removing in the short term where we don't have that requirement yet 🙂

Reply:
Happy to remove it if you think that's best, but it would anger sonar cloud and drop the coverage

Reply:
Very good point, maybe we just set a hard-coded screen value for this in the interim.
Just caught Sam (from analytics team) very quickly and he said that in the schema we'll be moving to, these title keys won't actually mean much so he's happy for us to put in a value for that 👍

Reply:
Sounds good, so for `screen` remove the enum value and write any appropriate string for the time being?